global_variable_generator.py

```diff
@@ -43,7 +47,7 @@ def main():
     constant_base_addr, constants = read_var_table(constant_table)
     shelf_base_addr, shelfs = read_var_table(shelf_table)
     sensor_base_addr, sensors = read_sensor_list_table(sensor_list_table)
-    sensor_data = read_sensor_data_table(sensor_data_table)
+    sens_base_addr, sensor_data = read_var_table(sensor_data_table)

```

Comment:
Instead of creating a dummy variable (sens_base_addr), you could also do...
`sensor_data = read_var_table(sensor_data_table)[1]`
to get the second component of the tuple.